docs/source/cli/xcube_chunk.md

```diff
@@ -6,21 +6,22 @@
 $ xcube chunk --help
 ```
     
-    Usage: xcube chunk [OPTIONS] <input> <output>
+    Usage: xcube chunk [OPTIONS] CUBE
     
-      (Re-)chunk dataset. Changes the external chunking of all variables of
-      <input> according to <chunks> and writes the result to <output>.
+      (Re-)chunk data cube. Changes the external chunking of all variables of
+      CUBE according to <CHUNKS> and writes the result to <OUTPUT>.
     
     Options:
-      -f, --format <format>  Format of the output. If not given, guessed from
-                             <output>.
-      -p, --params <params>  Parameters specific for the output format. Comma-
+      -o, --output <OUTPUT>  Output path. Defaults to 'out.zarr'
+      -f, --format <FORMAT>  Format of the output. If not given, guessed from
+                             <OUTPUT>.
+      --params <PARAMS>      Parameters specific for the output format. Comma-
                              separated list of <key>=<value> pairs.
-      -c, --chunks <chunks>  Chunk sizes for each dimension. Comma-separated list
+      --chunks <CHUNKS>      Chunk sizes for each dimension. Comma-separated list
```

Comment:
Either use `<option-arg>` or `OPTION-ARG`, but not the combination `<OPTION-ARG>`.

I vote for `OPTION-ARG`, because for arguments, we also use e.g. `CUBE` rather than `<CUBE>`.